src/subapps/admin/components/Settings/ViewsSubView.tsx

```diff
@@ -392,42 +435,72 @@ const ViewsSubView = () => {
           </Col>
         </Row>
 
-        <Table<TViewType>
+        <Table<SubView>
           loading={status === 'loading'}
           className="views-table"
           rowClassName="view-item-row"
           columns={columns}
-          dataSource={views?.results}
+          dataSource={views?.results as SubView[]}
```

Comment:
I might have overlooked something, but I think this type cast is technically incorrect. The `fetchViewsList` function returns a promise of an object where the `results` is *not* `SubView[]`, instead it is `Omit<Subview, 'indexingErrors'[]` (correct me if I'm wrong). Also, the `views?.results` can be `undefined` too. Therefore, the type casting might cause some bugs. 

I  would actually recommend to improve the type definition of this function instead of the explicit type casting here and on line [428](https://github.com/BlueBrain/nexus-web/pull/1448/files#diff-ce1ad27afb9d059ed2ab2fb4c9cf0a54ce5db0da412a4170d2be28afc06b11fdR428). So something like this can be added to the `fetchViewsList` function:

```typescript
const fetchViewsList = async ({
  nexus,
  orgLabel,
  projectLabel,
}: {
  nexus: NexusClient;
  orgLabel: string;
  projectLabel: string;
}): Promise<{                     // <----------------- Change
  errors: PromisePoolError<Omit<SubView, 'indexingErrors'>, any>[]; // <----------------- Change
  results: Omit<SubView, 'indexingErrors'>[]; // <----------------- Change
}> => {
```

After this, the type casting on both the lines should be removed (which might cause some issues with `undefined` which should be dealt with according to the situation).